### PR TITLE
Issue #9593: update example of AST for TokenTypes.UNARY_MINUS

### PR DESCRIPTION
**Pull request by gurleennsidhuu**
Issue #9593: update example of AST for TokenTypes.UNARY_MINUS
BEFORE:
be

AFTER:
af

### CODE REVIEW

**Review by AkMo3 — CHANGES_REQUESTED** [review]
Point:

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -919,6 +919,28 @@
...
+     *   +--UNARY_MINUS
+     *       |
+     *       +--EXPR
+     *          |
+     *          +--ASSIGN (=)
+     *              |
+     *              +--IDENT (a)
+     *              +--MINUS (-)
+     *                  |
+     *                  +--IDENT (b)
```
Comment (AkMo3) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:940]: @gurleennsidhuu Please share CLI output, and site output. Also, please match CLI output with your changes. These changes doesn't seem correct to me.

**Review by AkMo3 — CHANGES_REQUESTED** [review]
Point:

**Review by romani — CHANGES_REQUESTED** [review]
javadoc should use CLI output with minor corrections as defined in issue.
Please take a look in example PR.

**Comment by romani** [comment]
If no response in 5 days, I will close PR

**Pull request closed without merging**